scripts/release_scripts/deploy.py

```diff
@@ -455,9 +455,13 @@ def execute_deployment():
 
     install_third_party_libs.main()
 
-    if not common.is_current_branch_a_release_branch():
+    if common.is_current_branch_a_test_branch() and (
+            app_name == APP_NAME_OPPIASERVER):
+        raise Exception('Test branch cannot be deployed to prod.')
+    if not (common.is_current_branch_a_release_branch() or (
+            common.is_current_branch_a_test_branch())):
```

Comment:
Should we have this check before line 458? (Though it doesn't matter I was thinking like maybe we should first check the branch name is as expected and then the compatibility of the branch.)

Reply:
Done.